lib/web_ui/lib/src/engine/dom.dart

```diff
@@ -1390,6 +1392,106 @@ extension DomCSSRuleListExtension on DomCSSRuleList {
   external int get length;
 }
 
+/// Trusted Types API
+
+/*
+// Trusted Types API (TrustedTypePolicy, TrustedScript, TrustedScriptURL)
+// https://developer.mozilla.org/en-US/docs/Web/API/TrustedTypesAPI
```

Comment:
I like the idea of leaving links to MDN docs! I think these can be in the normal `///` style. Do we need to switch comment styles in the middle?

Reply:
These were more like a "chapter separator" in the middle of this humongous file. I can add `///` comments linking to the appropriate MDN pages in the js-interop classes themselves!